examples/nidaqmx_analog_input/pyproject.toml

```
@@ -6,7 +6,7 @@ authors = ["National Instruments"]
 
 [tool.poetry.dependencies]
 python = "^3.8"
-nidaqmx = { version = ">=0.8.0-dev0", extras = ["grpc"], allow-prereleases = true }
+nidaqmx = { version = ">=0.8.0", extras = ["grpc"], allow-prereleases = true }
```

We don't need prereleases anymore.
```suggestion
nidaqmx = { version = ">=0.8.0", extras = ["grpc"] }
```

---

Perhaps we should cherry-pick this change from `releases/1.1` to `main`. I didn't make it in `main` because it broke the "check examples" workflow, but now that 1.1.0 is released it won't break the "check examples" workflow.
https://github.com/ni/measurementlink-python/commit/6f79e8f07d853fb6e09b82bd5774edb3fedaec46

---

@bkeryan I have cherry picked this commit https://github.com/ni/measurementlink-python/commit/6f79e8f07d853fb6e09b82bd5774edb3fedaec46 and created a new PR https://github.com/ni/measurementlink-python/pull/340. Please refer.